src/questions/smart_contracts.yaml

```diff
@@ -0,0 +1,967 @@
+      key: A
+    - content: Including transaction
+      key: B
+    - content: Re-ordering transactions
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 8aeae442-78fd-4e64-98e7-859b6231c143
+  type: SingleChoice
+  content: What does Miners Extractable Value refer to?
+  hint: NoHint
+  explanation: Miner Extractable Value is the extra income other than standard block reward and gas fees which the miners get by tampering with the transaction orders (front-running).
+  answerKeys:
+    - D
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Standard block reward
+      key: A
+    - content: Gas fees
+      key: B
+    - content: Interest on large order
+      key: C
+    - content: Extra source of income from transactions
+      key: D
+- uuid: aa22fc95-3f12-4ff7-938b-5414e70a1370
+  type: MultipleChoice
+  content: How do “searchers” execute gas golfing? Gas golfing refers to programming transactions using the least amount of gas.
+  hint: Decrease the storage of a block
+  explanation: Addresses which have a long string of zeros reduce the size of the block and hence the gas required to mine it. Moreover, the contract must not be kept empty it should have some amount of ERC-20 to remove the gas required for initialising the contract.
+  answerKeys:
+    - A
+    - B
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "High"
+  choices:
+    - content: Using addresses which have long string of zeros.
+      key: A
+    - content: Leaving some amount of ERC-20 tokens in contracts.
+      key: B
+    - content: Reducing transactions into smaller parts.
+      key: C
+    - content: Use more efficient systems to execute transactions.
+      key: D
+- uuid: c646d255-f94f-43a4-89fd-54fa744001d3
+  type: MultipleChoice
+  content: Which of the following are examples of MEV?
+  hint: NoHint
+  explanation: DEX Arbitrage and Sandwich trading are the examples of MEV.
+  answerKeys:
+    - A
+    - C
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "High"
+  choices:
+    - content: DEX Arbitrage
+      key: A
+    - content: Selling of NFTs
+      key: B
+    - content: Sandwich trading
+      key: C
+    - content: None of the above
+      key: D
+- uuid: a97965ce-77f6-438d-8206-584ccd52ce49
+  type: MultipleChoice
+  content: What are the ill-effects of increasing MEV?
+  hint: Effets of MEV crowding
+  explanation: The ill-effects of MEV include increase in the gas fees and high MEV compared to standard block reward which would directly effect to the end user.
+  answerKeys:
+    - C
+    - D
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Miners get more money-minded
+      key: A
+    - content: More transactions occur on blockchain creating traffic.
+      key: B
+    - content: High gas fees
+      key: C
+    - content: Reduction in the number of transactions on the blockchain.
```

Comment:
I am not sure if `High gas fees` and `Reduction in the number of transactions on the blockchain.` are an effect of ill effects of MEV.

One of the ill effects of MEV I feel is that it can lead to unexpected exchange rates, as the transactions can be front running

https://coinmarketcap.com/alexandria/glossary/miner-extractable-value-mev

> One of the first forms of MEV that users suffered from was [front running](https://coinmarketcap.com/alexandria/glossary/front-running) done by bots that merely replicated users transactions with a higher gas price so that miners would pick their more expensive transactions over others. Although front running has evolved, and it's no longer represented in that form, miners are now taking advantage of such attacks. Front running occurs when miners are making a profit by placing their specific transaction right before the users, making the later one fail while the miners transaction is successful and profitable. This occurs because the miners are able to reorganize transactions at their own will, using their position of privileged information to execute a trade first by taking advantage of the trading opportunities the users transactions signal. 
> Alongside front running, the most common MEV attacks also include back running, which refers to miners making a profit by taking advantage of how the execution of the users transactions will change the market conditions and placing their specific transaction right after the users. 
> 
> A sandwich attack is the combination of the above two to specifically make a profit and take advantage of the users’ submitted transaction.

Reply:
Made the changes